package.json

```
@@ -13,7 +13,7 @@
     "format": "biome format",
     "format:write": "biome format --write",
     "lint": "biome lint",
```

Must've made an oversight on the lint command, there is no such thing as a "lint" command in Biome:

```suggestion
    "lint": "biome check",
```